src/rules/convertRules.ts

```diff
@@ -69,16 +73,15 @@ export const convertRules = (
                     ),
                 );
             } else {
-                const existingNotices = existingConversion.notices || [];
-                const newNotices = newConversion.notices || [];
-
                 converted.set(changes.ruleName, {
                     ...existingConversion,
                     ruleArguments: merger(
                         existingConversion.ruleArguments,
                         newConversion.ruleArguments,
                     ),
-                    notices: [...existingNotices, ...newNotices],
+                    notices: Array.from(
+                        new Set([...existingConversion.notices, ...newConversion.notices]),
```

Comment:
Let me know if this is better than it was previously (setting a default value for notices earlier on vs in the `existingNotices` and `newNotices` variables.

Reply:
@egiurleo unfortunately no, if `newNotices` is `undefined`, this will throw an error:

Trying out in a Node CLI:

```ts
> [...undefined, ...[]]
Thrown:
TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```

...which seems like a missing unit test!